www/src/components/shell/Terminal.js

```diff
@@ -161,21 +161,22 @@ export function Shell({ shell }) {
 
 export function Terminal() {
   const { data } = useQuery(CLOUD_SHELL_QUERY, { pollInterval: 5000, fetchPolicy: 'cache-and-network' })
+  const { shell } = data
+  const { alive } = data?.shell || false
+  const { status } = data?.shell || undefined
```

Comment:
`const { alive = false, status = undefined } = data?.shell` should do the same.

Do you need `undefined` here?

Reply:
That's why I've had to split it
![image](https://user-images.githubusercontent.com/2285385/182578449-f92ef879-e29f-415d-a23d-916ff53d86d4.png)